src/EventHandler/Media/GeoPoint.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace danog\MadelineProto\EventHandler\Media;
+
+use JsonSerializable;
+
+final class GeoPoint implements JsonSerializable
+{
+    /** @var int Longitude */
+    public readonly int $long;
+    /** @var int Latitude */
+    public readonly int $lat;
+    /** @var int 	Access hash */
+    public readonly int $accessHash;
+    /** @var int The estimated horizontal accuracy of the location, in meters; as defined by the sender. */
+    public readonly ?int $accuracyRadius;
+
+    public function __construct(array $rawGeoPoint)
+    {
+        $this->long = $rawGeoPoint['long'];
+        $this->lat = $rawGeoPoint['lat'];
+        $this->accessHash = $rawGeoPoint['accessHash'];
+        $this->accuracyRadius = $rawGeoPoint['accuracyRadius'] ?? null;
```

Comment:
accuracy_radius

Reply:
Its not good idea to use snack case for properties like you use chatId instead of chat_id snack case is ugly here